src/SFML/Window/WindowBase.cpp

```diff
@@ -68,13 +69,28 @@ WindowBase::WindowBase(WindowHandle handle)
 }
 
 
+////////////////////////////////////////////////////////////
+WindowBase::WindowBase(WindowBase&& other) noexcept :
+m_impl(std::exchange(other.m_impl, nullptr)),
+m_size(std::exchange(other.m_size, {}))
+{
+    if (&other == getFullscreenWindow())
+        setFullscreenWindow(this);
+    other.close();
+}
+
+
 ////////////////////////////////////////////////////////////
 WindowBase::~WindowBase()
 {
     WindowBase::close();
 }
 
 
+////////////////////////////////////////////////////////////
+WindowBase& WindowBase::operator=(WindowBase&&) noexcept = default;
```

Comment:
Does this need to include similar logic for setting the full screen window? I would assume so.